### ISSUE TITLE
babel-runtime

### ISSUE THREAD

**Issue by NathanRussellUK** [RIP21/react-simplemde-editor#72]
I've installed the package via NPM, but there seems to be a few missing dependencies - Unsure if I've missed a step somewhere...

Any help much appreciated!

```
ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/core-js/object/assign' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 13:14-60
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js

ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/core-js/object/entries' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 9:15-62
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js

ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/classCallCheck' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 17:23-70
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js

ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/inherits' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 25:17-58
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js

ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/possibleConstructorReturn' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 21:34-92
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js

ERROR in ./node_modules/react-simplemde-editor/lib/index.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/typeof' in '/home/nathan/code/bath-north-labour/node_modules/react-simplemde-editor/lib'
 @ ./node_modules/react-simplemde-editor/lib/index.js 5:15-54
 @ ./distribution/app/components/views/page.js
 @ ./distribution/app/components/app.js
 @ ./distribution/app/client.js
```

**Comment by NathanRussellUK** [comment]
Rolling back to 3.6.4 seemed to fix the issue - I believe I was using 3.6.16!

**Comment by LeonStaufer** [comment]
I have been having a similar issue. You can try installing the [Babel runtime and transform plugin](https://babeljs.io/docs/en/babel-plugin-transform-runtimel).

However, even with this, I was not able to fix it.

**Comment by RIP21** [comment]
Guys, I promise to fix all issues this weekend. Sorry for inconvenience, just too busy to touch it. Thanks for understanding and thanks for reporting an issue.

**Comment by LeonStaufer** [comment]
No worries, I've circumvented the issue by creating a custom wrapper for React.

I am assuming the error is because of a change in Babel 7.

**Comment by RIP21** [comment]
This is fixed in `v3.6.17`, also typescript typings added ;)
